lib/libvfio-user.c

```diff
@@ -587,7 +588,8 @@ handle_dma_unmap(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg,
         msg->out_iovecs->iov_len = region->bitmap->size;
```

Comment:
don't we need the reply to have the same vfio_user_dma_region header? So it can specify the actual needed argsz?

Reply:
I don't think so, the client knows the size since it's the one that sets `region->bitmap->size` in the first place, so it knows that `argsz` but be at least `sizeof(strcut vfio_user_bitmap) + region->bitmap->size`.

Reply:
I don't expect it to be common to use these argsz semantics, but I also don't see a good reason to differ from the expectations of the spec and the current code elsewhere. We should respond